Documentation/CRDs/Block-Storage/ceph-block-pool-crd.md

```diff
@@ -205,7 +205,7 @@ stretched) then you will have 2 replicas per datacenter where each replica ends
   require names that are incompatible with K8s resource names. These special pools can be configured
   by setting this `name` to override the name of the Ceph pool that is created instead of using the `metadata.name` for the pool.
   Only the following pool names are supported: `device_health_metrics`, `.nfs`, and `.mgr`. See the example
-  [builtin mgr pool](https://github.com/rook/rook/blob/master/deploy/examples/pool-builtin-mgr.yaml).
+  [builtin mgr pool](https://github.com/rook/rook/blob/release-1.10/deploy/examples/pool-builtin-mgr.yaml).
```

Comment:
I changed a lot of these links to example manifests. It would be pretty easy to write a `sed` replace script for this that we could run in CI to ensure the branch name is reflected in the docs. I'll do that quickly.

Reply:
Wait, the docs publish is supposed to handle this automatically. @galexrt can confirm, or you can check in the release-1.9 published docs to see that they're correct.

Reply:
I checked, and it's all good. I forgot about that detail, thanks